tests/test_digital.py

```diff
@@ -150,12 +151,12 @@ def test_CRY_state01_controlqubit_0() -> None:
 
 
 def test_CSWAP_state101_controlqubit_0() -> None:
-    result: Tensor = pyq.CSWAP((0, 1), 2)(product_state("101"), None)
-    assert torch.allclose(product_state("110"), result)
+    result: Tensor = pyq.CSWAP(2, (0, 1))(product_state("101"), None)
+    assert torch.allclose(product_state("011"), result)
 
 
 def test_CSWAP_state110_controlqubit_0() -> None:
-    result: Tensor = pyq.CSWAP((0, 1), 2)(product_state("101"), None)
+    result: Tensor = pyq.CSWAP(2, (0, 1))(product_state("110"), None)
```

Comment:
This test can just be removed, same as the one before it. Just keep the one parameterized over various input, output states.

Reply:
Removed.